department-of-veterans-affairs/caseflow
# Adds schedule_periods

Connects #5709 

### Description
This PR adds a new table: schedule_periods.

### Testing Plan
1. Run the migration
1. Run 'RoSchedulePeriod.new(user: User.first, start_date: Date.today, end_date: Date.today).save!' in the console
1. Run 'JudgeSchedulePeriod.new(user: User.first, start_date: Date.today, end_date: Date.today).save!' in the console
1. Confirm they both show up when running SchedulePeriod.all
